README.md

```diff
@@ -12,6 +12,8 @@ Custom component for lovelace to be used as a panel for viewing security cameras
 | cameras | list | _See camera section below_ | **Required**
 | thumb_interval | number | Update interval for thumbnails in seconds (_min_ 0.5) | 10
 | update_interval | number | Update interval for main image in seconds (_min_ 0.5) | 1
+| show_capture_buttons | boolean | Show screenshot and record buttons | true
+| recording_duration | number | Number of seconds to record after clicking record button (_min_ 0.5) | 10
```

Comment:
I'm wondering if this should be an object instead. because you might want to have a future option to record on option. Thoughts